### DISCUSSION TITLE
Information output when recognizing Yolo v8

### DISCUSSION THREAD

**Discussion by Hynker** [ultralytics/ultralytics#1349] [Q&A] [answered]
How can I remove the output of recognition information to the console when using YOLOv8?
![1](https://user-images.githubusercontent.com/82664284/224069912-3017f391-e64f-4a67-88f9-ee619e6526f0.png)

**Comment by luislortega** [comment]
what up buddy!!

this will fix your problem

`import logging

    logger = logging.getLogger()
    logger.setLevel(logging.INFO)
    formatter = logging.Formatter('%(asctime)s - %(name)s - %(levelname)s - %(message)s')
    handler = logging.StreamHandler(sys.stdout)
    handler.setFormatter(formatter)
    logger.addHandler(handler)
    logging.getLogger("ultralytics").setLevel(logging.WARNING)`

if works, mark me as answer ;)

**Reply by glenn-jocher** [reply]
@luislortega hello!

To reduce the verbosity of the console output when using YOLOv8, you can adjust the logging level in your code. By default, YOLOv8 may print various informational messages to the console during operation. To change this behavior, you can set the logging level to a higher threshold such as `WARNING` or `ERROR` to only display more critical messages.

Please refer to the Ultralytics documentation for more details on how to configure logging within the YOLOv8 environment. The documentation provides guidance on customizing different aspects of the model's behavior, including output verbosity.

If you need further assistance, feel free to reach out again. Happy coding! 😊

**Comment by KatarinaPopikova** [comment, marked as answer]
Try set atribute verbose to False ("verbose=False")

For example:
model.predict(source=SOURCE, verbose=False)

**Reply by glenn-jocher** [reply]
@KatarinaPopikova to suppress the recognition information output to the console when using YOLOv8, you can indeed set the `verbose` attribute to `False` when calling the predict function. This will reduce the amount of information printed to the console during prediction. If you need further guidance on how to use this attribute, please refer to the Predict mode documentation in the Ultralytics Docs. 😊🔍

**Comment by fuleige** [comment]
import ultralytics.utils as ultralytics_utils
import logging

ultralytics_utils.LOGGER.setLevel(logging.WARN)

**Reply by glenn-jocher** [reply]
@fuleige to suppress recognition information output to the console when using YOLOv8, you can adjust the logging level as follows:

```python
import ultralytics.utils as ultralytics_utils
import logging

ultralytics_utils.LOGGER.setLevel(logging.WARN)
```

This will limit the log messages to warnings and above, reducing the verbosity of the console output.
